src/packages/client/src/__tests__/integration/happy/raw-transactions/test.ts

```diff
@@ -295,7 +295,6 @@ describe('transaction', () => {
       ]),
       prisma.$transaction([prisma.user.findFirst(), prisma.post.findFirst()]),
     ])
-    await new Promise((res) => setTimeout(res, 200))
```

Comment:
Why did we remove this? Why was it even here?

Reply:
Was an attempt to fix to logging flakynes